smart-contracts/contracts/cl-vault/src/test_tube/initialize.rs

```diff
@@ -144,7 +140,7 @@ pub mod initialize {
     ) {
         init_test_contract_with_dex_router_and_swap_pools(
             "./test-tube-build/wasm32-unknown-unknown/release/cl_vault.wasm",
-            "../../artifacts/cw_dex_router.wasm",
+            "../../artifacts/dex_router_osmosis.wasm",
```

Comment:
I guess to properly work in the ci, we need to build that with the test-tube-build and read it from there